pkg/controller/atlasproject/teams.go

```diff
@@ -1,10 +1,13 @@
 package atlasproject
 
 import (
-	"go.mongodb.org/atlas-sdk/v20231115008/admin"
+	"fmt"
+
 	"k8s.io/apimachinery/pkg/types"
 	controllerruntime "sigs.k8s.io/controller-runtime"
 
+	"github.com/mongodb/mongodb-atlas-kubernetes/v2/internal/translation/teams"
```

Comment:
move package import below

Reply:
To give more context, we should standardise around just 3 groups of imports max, in order:

1. stdlib
2. other deps
3. ako internal deps 

`goimports` will respect our groupings and reorder each group alphabetically.

Sometimes when you change something, a new line is added and we end up with more groups than needed, and the go formatter does not fix it for us.